.gitignore

```diff
@@ -2,3 +2,4 @@
 data/raw_json/**/*
 data/clean_data/**/*
 models/word2vec/*
+img/**/*
```

Comment:
If you want to remove the currently tracked images you need to run:

`git rm -r --cached img/*`

Otherwise git is still going to track the images that were already made